docs/maintain-guides-how-to-setup-a-validator-with-reverse-proxy.md

```diff
@@ -0,0 +1,183 @@
+---
+id: maintain-guides-how-to-setup-a-validator-with-reverse-proxy
+title: Set Up a Validator with NGINX Reverse Proxy
+sidebar_label: Set Up a Validator with NGINX Reverse Proxy
+---
+
+This guide assumes that you have already configured your hardware with the appropriate specs. It has same configuration as the
+[polkadot validator setup](https://github.com/w3f/polkadot-secure-validator).
+
+We will walk you through how to configure a reverse proxy using NGINX in front of your validator node. The
+validator uses the reverse proxy only to talk to the public facing nodes, to protect it from certain types of attacks.
+
+### 1. Firewall configuration
+
+We will configure the firewall with [ufw](https://wiki.ubuntu.com/UncomplicatedFirewall). There needs to be three main ports for this setup. 
+- An SSH port, commonly ssh/tcp port `22`.
+- A proxy port 
+- p2p port: must be denied at the firewall level. 
+
+In this example, we will assign the port number `2435` to the proxy port and the port number `30333` to the p2p port. To enable the firewall and the use of the ports, allow SSH access.
+
```

Comment:
Would add a note here something like:

> NOTE: For parachains you will need to allow for both inbound and outbound traffic on the P2P port. Since the proxy port is the public facing port, this will need to have inbound and outbound traffic open, with the normal p2p port closed.

Reply:
Added.